frontend/src/i18n/locales/types.ts

```diff
@@ -1,23 +1,49 @@
-import { Navbar, Common, Login, PublicForm, Fields, HeaderAndInstructions, Logic, ThankYou} from './features'
+import {
+  Common,
+  Fields,
+  HeaderAndInstructions,
+  Logic,
+  Login,
+  Navbar,
+  PublicForm,
+  ThankYou,
+} from './features'
 
 interface Translation {
   translation: {
     features: {
-      navbar?: Navbar
+      adminForm?: {
+        sidebar?: {
+          fields?: Fields
+          headerAndInstructions?: HeaderAndInstructions
+          logic?: Logic
+          thankYou?: ThankYou
+        }
+        navbar?: Navbar
+      }
       common?: Common
       publicForm?: PublicForm
       login?: Login
-      fields?: Fields
-      headerAndInstructions?: HeaderAndInstructions
-      logic?: Logic
-      thankYou?: ThankYou
     }
   }
 }
 
 export interface FallbackTranslation extends Translation {
   translation: {
-    features: Required<Translation['translation']['features']>
```

Comment:
Could add a new utility type "DeepRequired" which recursively makes the entire directory required?